ttnn/cpp/ttnn/operations/eltwise/unary/unary_pybind.hpp

```diff
@@ -709,182 +708,186 @@ void bind_unary_composite(py::module& module, const unary_operation_t& operation
 
-            Applies {0} to :attr:`input_tensor` element-wise.
+        {8}
 
-            .. math::
-                {0}(\\mathrm{{input\\_tensor}}_i)
+        Args:
+            * :attr:`input_tensor`
 
-            Args:
-                * :attr:`input_tensor`
-                * :attr:`scale`
-                * :attr:`alpha`
+        Keyword args:
+            * :attr:`{2}` (float): {3} , Default value = {4}
+            * :attr:`{5}` (float): {6} , Default value = {7}
+            * :attr:`memory_config` [ttnn.MemoryConfig]: memory config for the output tensor
 
-            Keyword Args:
-                * :attr:`memory_config` (Optional[ttnn.MemoryConfig]): Memory configuration for the operation.
+        Example:
 
-            Example:
-
-                >>> tensor = ttnn.from_torch(torch.tensor((1, 2), dtype=torch.bfloat16), device=device)
-                >>> output = {1}(tensor)
+            >>> tensor = ttnn.from_torch(torch.tensor((1, 2), dtype=torch.bfloat16), device=device)
+            >>> output = {1}(tensor, {2} = {4}, {5} = {7})
         )doc",
         operation.base_name(),
-        operation.python_fully_qualified_name());
+        operation.python_fully_qualified_name(),
+        parameter_name_a,
+        parameter_a_doc,
+        parameter_a_value,
+        parameter_name_b,
+        parameter_b_doc,
+        parameter_b_value,
+        description);
 
     bind_registered_operation(
         module,
         operation,
         doc,
         ttnn::pybind_overload_t{
             [](const unary_operation_t& self,
-               const Tensor& input_tensor,
-               float scale,
-               float alpha,
-               const std::optional<MemoryConfig>& memory_config) {
-                    return self(input_tensor, scale, alpha, memory_config);
-                },
+               const ttnn::Tensor& input_tensor,
+               float parameter_a,
+               float parameter_b,
+               const std::optional<MemoryConfig>& memory_config)  {
+                return self(input_tensor, parameter_a, parameter_b, memory_config);
+            },
             py::arg("input_tensor"),
-            py::arg("scale")= 1.0507009873554804934193349852946,
-            py::arg("alpha")= 1.6732632423543772848170429916717,
             py::kw_only(),
+            py::arg(parameter_name_a.c_str()) = parameter_a_value,
+            py::arg(parameter_name_b.c_str()) = parameter_b_value,
             py::arg("memory_config") = std::nullopt});
 }
-
+//OpHandler_two_float_with_default
 template <typename unary_operation_t>
-void bind_unary_composite_threshold_value(py::module& module, const unary_operation_t& operation) {
+void bind_unary_composite_one_int(py::module& module, const unary_operation_t& operation, const std::string& parameter_name_a, const std::string& parameter_a_doc, const std::string& description) {
     auto doc = fmt::format(
```

Comment:
Again, what is wrong with the original names? Didn't they communicate the feature intent of the function?